Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -4125,7 +4125,8 @@ void FrameLoader::continueLoadAfterNavigationPolicy(const ResourceRequest& reque
         return;
     }
 
-    if (auto pendingDispatchNavigateEvent = m_policyDocumentLoader->triggeringAction().takePendingDispatchNavigateEvent()) {
+    if (m_policyDocumentLoader) {
+        auto pendingDispatchNavigateEvent = m_policyDocumentLoader->triggeringAction().takePendingDispatchNavigateEvent();
         if (!pendingDispatchNavigateEvent())
```

Comment:
This should be done only if pendingDispatchNavigateEvent is non-empty.  I think this would be a better change:

if (auto pendingDispatchNavigateEvent = m_policyDocumentLoader ? m_policyDocumentLoader->triggeringAction().takePendingDispatchNavigateEvent() : { })

Reply:
Compile error: `initializer list cannot be used on the right hand side of operator ':'`; I will change it to:
```
if (m_policyDocumentLoader) {
     auto pendingDispatchNavigateEvent = m_policyDocumentLoader->triggeringAction().takePendingDispatchNavigateEvent();
     if (pendingDispatchNavigateEvent && !pendingDispatchNavigateEvent())
          return;
}
```

Reply:
Replacing the empty initializer list with std::function() would do it too